portfolio/src/main/webapp/script.js

```diff
@@ -30,7 +30,43 @@ function findZoe() {
   const location = locations[Math.floor(Math.random() * locations.length)];
 
   // Add it to the page.
-  const link = document.getElementById("link");
+  const link = document.getElementById('link');
   link.innerText = location;
   link.href = locationToLink[location];
 }
+
+function grow() {
+  const srcToAlt = {
+    'images/basil1.jpg': 'Basil Day 0',
+    'images/basil2.jpg': 'Basil Day 10',
+    'images/basil3.jpg': 'Basil Day 20',
+    'images/basil4.jpg': 'Basil Day 30',
+    'images/basil5.jpg': 'Basil Day 40',
+    'images/basil6.jpg': 'Basil Day 60',
+  };
+  const image = document.getElementById('basil');
+  var imgIndex = 1;
+  var imgSrc = 'images/basil' + imgIndex + '.jpg';
+  console.log(`Let's grow some basil`);
+
+  // Recursive call
+  function someTimeLater() {
+    setTimeout(function() {
+      console.log('Some time has passed ...');
+      // Change the image
+      imgSrc = 'images/basil' + imgIndex + '.jpg';
+      image.src = imgSrc;
+      image.alt = srcToAlt[imgSrc];
+      image.width = '400';
+      image.height = '400';
+      imgIndex++;
+      if (imgIndex <= 6) {
+        someTimeLater(); // Keep growing
+      } else {
+        alert(`This's the latest photo I have! The basil will keep growing ...`);
```

Comment:
you might want to put this in the text of the page instead of an alert box

Reply:
Done! I created a text box to hold the complete message. See this [commit](https://github.com/chenzizhao/step/pull/3/commits/e5ff317037857bd0a9e93550a4d8c6e9c8257dc8) .